src/Elasticsearch/ClientBuilder.php

```diff
@@ -0,0 +1,351 @@
+<?php
+
+namespace SphereMall\MS\Elasticsearch;
+
+use Elasticsearch\Client;
+use Elasticsearch\ConnectionPool\Selectors\RoundRobinSelector;
+use Elasticsearch\ConnectionPool\StaticNoPingConnectionPool;
+use Elasticsearch\Connections\ConnectionFactory;
+use Elasticsearch\Transport;
+use Psr\Log\NullLogger;
+
+/**
+ * Class ClientBuilder
+ *
+ * @category Elasticsearch
+ * @package  Elasticsearch\Common\Exceptions
+ * @author   Zachary Tong <zach@elastic.co>
+ * @license  http://www.apache.org/licenses/LICENSE-2.0 Apache2
+ * @link     http://elastic.co
+ */
+class ClientBuilder extends \Elasticsearch\ClientBuilder
```

Comment:
Do not rewrite the entire class. Use the programming patterns of the reassembly of private entities